src/java.base/share/classes/sun/nio/fs/Util.java

```diff
@@ -37,8 +37,10 @@
 class Util {
     private Util() { }
 
-    private static final Charset jnuEncoding = Charset.forName(
-        GetPropertyAction.privilegedGetProperty("sun.jnu.encoding"));
+    private static final Charset jnuEncoding =
+        Charset.forName(
+            GetPropertyAction.privilegedGetProperty("sun.jnu.encoding"),
+            sun.nio.cs.UTF_8.INSTANCE);
```

Comment:
Startup will ensure that sun.jnu.encoding has a value that is a supported encoding so I assume the the fallback in sun.nio.ch.Util is no longer needed.

Reply:
Reverted the fix.